tests/installation/welcome.pm

```diff
@@ -34,6 +34,7 @@ sub run() {
             send_key_until_needlematch 'scc-invalid-url-deleted', 'backspace';
             type_string get_var('SCC_URL_VALID');
             wait_still_screen 2;
+            save_screenshot;     # create a needle from this, then replace to an assert_screen
```

Comment:
With your approach I will break the test by default with a non-existing needle.
With my approach I get less stress. I can patiently wait for the screenshot.